### ISSUE TITLE
give some error message when using bad directive

### ISSUE THREAD

**Issue by bjornregnell** [VirtusLab/scala-cli#1636]
**Version(s)**
0.1.18

**Describe the bug**
scala-cli should complain if I write bad directives like by mistake replacing `using` with `use` after `//>`

**To Reproduce**
```
$ cat main.scala 
//> use scala "3.2.1"
//> use lib "org.jline:jline:3.21.0"

val t = org.jline.terminal.TerminalBuilder.terminal

@main def run = 
  println(s"hello $t")

$ scala-cli run main.scala 
Compiling project (Scala 3.2.1, JVM)
[error] ./main.scala:4:9: value jline is not a member of org
[error] val t = org.jline.terminal.TerminalBuilder.terminal
[error]         ^^^^^^^^^
Error compiling project (Scala 3.2.1, JVM)
Compilation failed
```
Above nothing is said about that the directive is bad. Instead it goes ahead and compiles and the compile error is not so helpful...

**Expected behaviour**
I should get some feedback that `use` is not a valid directive; it would be nice if it proposed something like `did you mean 'using'`  .

**Comment by Gedochao** [comment]
Hmm... I'm not sure about an error message. `//>` is still a comment, so I guess it should compile successfully, if it can't be parsed as a using directive...
But yeah, we definitely could print a warning and potentially maybe add an actionable diagnostic.
thoughts?

**Comment by bjornregnell** [comment]
Yes, warning and actionable diagnostics is the right way. And I agree that it should go ahead and try to compile despite the problem -- but in this case it will not compile as there is a missing dependency. I think scala-cli can and should help with some good output here.

**Comment by tgodzik** [comment]
@lwronski this would be a pretty neat actionable diagnostic. We should get some levenstein going maybe, which might be helpful in more cases.

**Comment by bjornregnell** [comment]
See also https://github.com/VirtusLab/scala-cli/discussions/266#discussioncomment-5087148

**Comment by SethTisue** [comment]
I guess this is under the same umbrella. I (and @Izmar, iirc — I may have the wrong user name, apologies if so) were surprised by the following behavior:

```
% cat S.scala 
//> using option -deprecation
class C
//> using scala 2.13.12
class D

% scala-cli compile S.scala
Compiling project (Scala 3.3.0, JVM)
Compiled project (Scala 3.3.0, JVM)
```

Note that the second directive was silently ignored. I expected either that the directives could go anywhere, _or_ that I would get a warning.
